recipes/jupyterlab_code_formatter/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "jupyterlab_code_formatter" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/ryantam626/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 923efce4fb9bae3196218624bb7e5e3f0f064b73eda01ef2cd60b9ad307beeaf
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  build:
+    - nodejs
+  host:
+    - jupyterlab
+    - pip
+    - python
+  run:
+    - notebook
+    - python >=3.6
```

Comment:
Should the run requirements be changed to jupyterlab instead of notebook and also contain nodejs to reflect the run time requirements of the javascript/frontend part?